doc/guide/cert-authentication.xml

```diff
@@ -63,6 +54,54 @@ pkcs15-init --store-private-key alice.p12 --format pkcs12 --auth-id 01
 
   </section>
 
+  <section id="certauth-server-ipa">
+    <title>Certificate import/mapping with FreeIPA</title>
+
+    <para>The domain's users get associated to certificates with
+      the <command>ipa user-add-cert</command> command. See the
+      <ulink url="https://www.freeipa.org/page/V4/User_Certificates#Feature_Management">
+      FreeIPA User Certificates documentation</ulink>:</para>
+
+<programlisting>
+# FreeIPA only accepts DER format, convert it
+openssl x509 -outform der -in alice.pem -out alice.der
+
+# assign it to the domain-managed user alice
+ipa user-add-cert alice --certificate="$(base64 alice.der)"
+</programlisting>
+
+  </section>
+
+  <section id="certauth-server-local">
+    <title>Certificate mapping to local user</title>
+
+    <para>If you do not want to use identity management, it is also possible to use
+      <ulink url="https://sssd.io/docs/design_pages/certmaps_for_LDAP_AD_file.html">custom certificate mapping rules</ulink>
+      in <citerefentry><refentrytitle>sssd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+      in a <code>[certmap/domain/user]</code> section, usually for the default <code>implicit_files</code>
+      pseudo-domain. For example, this matches a certificate with a particular Subject which is
+      signed by a particular CA, and maps it to the user <code>alice</code>:</para>
+
+<programlisting>
+[certmap/implicit_files/alice]
+matchrule = &lt;ISSUER&gt;^CN=CA,O=MyCompany$&lt;SUBJECT&gt;^DC=COM,DC=EXAMPLE,CN=alice$
```

Comment:
documenting `ISSUER` is fairly pointless, as it's fungible, and the cert needs to be signed by a given CA anyway.

Reply:
Fixed.